app/views/api/ads/index.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
I'd add this in `config/initializers/jbuilder.rb`:

```rb
Jbuilder.key_format camelize: :lower
```

So that field names convert automatically from (for example): `zip_code` to `zipCode`.

Wdyt?

You can also add it here per file instead of globally:

```rb
json.key_format! camelize: :lower
```

Reply:
I prefer `zip_code` but do not have strong opinion about it. Why do you mean `zipCode` is better?

Reply:
Hm, looks like we do not have any keys that would be affected by this :)

Reply:
...yet :)

It's more common to sue camelCase in JSON / Javascript

Reply:
I think our API will most probably not even be consumed by Javascript so I will leave it for now. Let's see if we get some more votes :)

Reply:
I don't think it's particularly beneficial to go either way. Depending on which language is consuming it, either one can feel nicer to the consumer so it's a coin toss.

Whatever we choose the only important thing is that we're consistent.

Reply:
Agreed, let's just keep it like this and be consistent :)